crates/solidity/testing/utils/src/cst_snapshots/test_nodes.rs

```diff
@@ -24,21 +24,19 @@ struct TestNodeBuilder {
 }
 
 impl Visitor<()> for TestNodeBuilder {
-    fn enter_rule(
+    fn rule_enter(
```

Comment:
I wonder why the change from `enter_rule()` to `rule_enter()`?

Reply:
Because I removed the `enter||exit_` prefix from `token`, and I wanted the method names in the `Visitor` to start with the type name i.e. `rule|token` for regularity.